deployment_test.go

```diff
@@ -83,3 +83,12 @@ func TestDeleteDeployment(t *testing.T) {
 	assert.Equal(t, id.DeploymentID, "0b1467fc-d5cd-4bbc-bac2-2805351cee1e")
 	assert.Equal(t, id.Version, "2014-08-26T08:20:26.171Z")
 }
+
+func TestDeleteDeploymentForce(t *testing.T) {
+	endpoint := newFakeMarathonEndpoint(t, nil)
+	defer endpoint.Close()
+	resp, err := endpoint.Client.DeleteDeployment(fakeDeploymentID, true)
+	assert.NoError(t, err)
```

Comment:
Can you please use `require.NoError` here? I don't think there's any value in checking `resp` if we have already received an error at this point.